loquacious-credenza/Lantern
# React native redux scaffolding

## Talk to me before merging this!!
#### What's this PR do?

This adds redux functionality to our react-native project.  It moves store configuration to root level of app and breaks out reducers and and actions into respective files.
#### What are the important parts of the code?

take a look at the simplifed `index.ios.js` file, `app.js`.  Currently there is one proof with `currentLocation` that is currently working. Much of the scaffolding has been created.
#### How should this be tested by the reviewer?

Download the branch.
`npm run project_setup`
`npm install`
`npm start`
go to xcode and run simulation.  you should see the app getting geolocation data.
#### Is any other information necessary to understand this?

The next steps are for the app to have routing incorporated.

closes #36
